### ISSUE TITLE
DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.

### ISSUE THREAD

**Issue by idkman2021** [OpenTabletDriver/OpenTabletDriver#2449]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### Steps to Reproduce

tablet works in fedora
open osu
it has built in OpenTabletDriver (people said I need to create bug report here and not in osu)
it says
`DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.`
and when I use tablet it just hides the cursor and doesn't work

### Expected Behavior

tablet works in osu

### Observed Behavior

tablet doesnt work in osu and instead hides the cursor

### Tablet Device

Wacom CTL-472

### Diagnostic Information

```
2022-10-07 09:12:58 [verbose]: ----------------------------------------------------------
2022-10-07 09:12:58 [verbose]: runtime Log for bigchungus (LogLevel: Verbose)
2022-10-07 09:12:58 [verbose]: Running osu 2022.923.1.0 on .NET 6.0.5
2022-10-07 09:12:58 [verbose]: Environment: Linux (Unix 5.19.11.201), 8 cores
2022-10-07 09:12:58 [verbose]: ----------------------------------------------------------
2022-10-07 09:12:58 [verbose]: Connected MIDI device: Midi Through Port-0
2022-10-07 09:12:58 [verbose]: GL Initialized
2022-10-07 09:12:58 [verbose]: GL Version:                 4.6 (Compatibility Profile) Mesa 21.3.9 (git-78c96ae5b6)
2022-10-07 09:12:58 [verbose]: GL Renderer:                Mesa Intel(R) UHD Graphics (CML GT2)
2022-10-07 09:12:58 [verbose]: GL Shader Language version: 4.60
2022-10-07 09:12:58 [verbose]: GL Vendor:                  Intel
2022-10-07 09:12:58 [verbose]: GL Extensions:              GL_ARB_multisample GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_copy_texture GL_EXT_subtexture GL_EXT_texture_object GL_EXT_vertex_array GL_EXT_compiled_vertex_array GL_EXT_texture GL_EXT_texture3D GL_IBM_rasterpos_clip GL_ARB_point_parameters GL_EXT_draw_range_elements GL_EXT_packed_pixels GL_EXT_point_parameters GL_EXT_rescale_normal GL_EXT_separate_specular_color GL_EXT_texture_edge_clamp GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_ARB_framebuffer_sRGB GL_ARB_multitexture GL_EXT_framebuffer_sRGB GL_IBM_multimode_draw_arrays GL_IBM_texture_mirrored_repeat GL_3DFX_texture_compression_FXT1 GL_ARB_texture_cube_map GL_ARB_texture_env_add GL_ARB_transpose_matrix GL_EXT_blend_func_separate GL_EXT_fog_coord GL_EXT_multi_draw_arrays GL_EXT_secondary_color GL_EXT_texture_env_add GL_EXT_texture_filter_anisotropic GL_EXT_texture_lod_bias GL_INGR_blend_func_separate GL_NV_blend_square GL_NV_light_max_exponent GL_NV_texgen_reflection GL_NV_texture_env_combine4 GL_S3_s3tc GL_SUN_multi_draw_arrays GL_ARB_texture_border_clamp GL_ARB_texture_compression GL_EXT_framebuffer_object GL_EXT_texture_compression_s3tc GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_MESA_window_pos GL_NV_packed_depth_stencil GL_NV_texture_rectangle GL_ARB_depth_texture GL_ARB_occlusion_query GL_ARB_shadow GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_mirrored_repeat GL_ARB_window_pos GL_ATI_fragment_shader GL_EXT_stencil_two_side GL_EXT_texture_cube_map GL_NV_copy_depth_to_color GL_NV_depth_clamp GL_NV_fog_distance GL_NV_half_float GL_APPLE_packed_pixels GL_ARB_draw_buffers GL_ARB_fragment_program GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_vertex_program GL_ARB_vertex_shader GL_ATI_draw_buffers GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_MESA_pack_invert GL_NV_primitive_restart GL_ARB_depth_clamp GL_ARB_fragment_program_shadow GL_ARB_half_float_pixel GL_ARB_occlusion_query2 GL_ARB_point_sprite GL_ARB_shading_language_100 GL_ARB_sync GL_ARB_texture_non_power_of_two GL_ARB_vertex_buffer_object GL_ATI_blend_equation_separate GL_EXT_blend_equation_separate GL_OES_read_format GL_ARB_color_buffer_float GL_ARB_pixel_buffer_object GL_ARB_texture_compression_rgtc GL_ARB_texture_float GL_ARB_texture_rectangle GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_shared_exponent GL_ARB_framebuffer_object GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_packed_depth_stencil GL_ARB_vertex_array_object GL_ATI_separate_stencil GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_gpu_program_parameters GL_EXT_gpu_shader4 GL_EXT_texture_array GL_EXT_texture_integer GL_EXT_texture_sRGB_decode GL_EXT_timer_query GL_OES_EGL_image GL_AMD_performance_monitor GL_EXT_texture_buffer_object GL_AMD_texture_texture4 GL_ARB_copy_buffer GL_ARB_depth_buffer_float GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_map_buffer_range GL_ARB_texture_buffer_object GL_ARB_texture_rg GL_ARB_texture_swizzle GL_ARB_vertex_array_bgra GL_EXT_texture_swizzle GL_EXT_vertex_array_bgra GL_NV_conditional_render GL_AMD_conservative_depth GL_AMD_depth_clamp_separate GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_ARB_ES2_compatibility GL_ARB_blend_func_extended GL_ARB_compatibility GL_ARB_debug_output GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_explicit_attrib_location GL_ARB_fragment_coord_conventions GL_ARB_provoking_vertex GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_shader_stencil_export GL_ARB_shader_texture_lod GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_multisample GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_uniform_buffer_object GL_ARB_vertex_type_2_10_10_10_rev GL_EXT_provoking_vertex GL_EXT_texture_snorm GL_MESA_texture_signed_rgba GL_NV_copy_image GL_NV_texture_barrier GL_ARB_draw_indirect GL_ARB_get_program_binary GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_robustness GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_precision GL_ARB_shader_subroutine GL_ARB_texture_compression_bptc GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_viewport_array GL_EXT_direct_state_access GL_EXT_vertex_attrib_64bit GL_AMD_multi_draw_indirect GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_base_instance GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_internalformat_query GL_ARB_map_buffer_alignment GL_ARB_shader_atomic_counters GL_ARB_shader_image_load_store GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_texture_storage GL_ARB_transform_feedback_instanced GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_transform_feedback GL_AMD_query_buffer_object GL_AMD_shader_trinary_minmax GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_clear_buffer_object GL_ARB_compute_shader GL_ARB_copy_image GL_ARB_explicit_uniform_location GL_ARB_fragment_layer_viewport GL_ARB_framebuffer_no_attachments GL_ARB_invalidate_subdata GL_ARB_multi_draw_indirect GL_ARB_program_interface_query GL_ARB_robust_buffer_access_behavior GL_ARB_shader_image_size GL_ARB_shader_storage_buffer_object GL_ARB_stencil_texturing GL_ARB_texture_buffer_range GL_ARB_texture_query_levels GL_ARB_texture_storage_multisample GL_ARB_texture_view GL_ARB_vertex_attrib_binding GL_KHR_debug GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_AMD_pinned_memory GL_ARB_buffer_storage GL_ARB_clear_texture GL_ARB_compute_variable_group_size GL_ARB_enhanced_layouts GL_ARB_indirect_parameters GL_ARB_internalformat_query2 GL_ARB_multi_bind GL_ARB_query_buffer_object GL_ARB_seamless_cubemap_per_texture GL_ARB_shader_draw_parameters GL_ARB_shader_group_vote GL_ARB_shading_language_include GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_stencil8 GL_ARB_vertex_type_10f_11f_11f_rev GL_EXT_shader_framebuffer_fetch GL_EXT_shader_integer_mix GL_INTEL_performance_query GL_ARB_ES3_1_compatibility GL_ARB_clip_control GL_ARB_conditional_render_inverted GL_ARB_cull_distance GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_get_texture_sub_image GL_ARB_pipeline_statistics_query GL_ARB_shader_texture_image_samples GL_ARB_texture_barrier GL_ARB_transform_feedback_overflow_query GL_EXT_polygon_offset_clamp GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_context_flush_control GL_KHR_robust_buffer_access_behavior GL_ARB_ES3_2_compatibility GL_ARB_fragment_shader_interlock GL_ARB_gpu_shader_int64 GL_ARB_parallel_shader_compile GL_ARB_post_depth_coverage GL_ARB_shader_atomic_counter_ops GL_ARB_shader_ballot GL_ARB_shader_clock GL_ARB_shader_viewport_layer_array GL_EXT_shader_samples_identical GL_EXT_texture_sRGB_R8 GL_KHR_no_error GL_KHR_texture_compression_astc_sliced_3d GL_NV_fragment_shader_interlock GL_ARB_gl_spirv GL_ARB_spirv_extensions GL_MESA_shader_integer_functions GL_ARB_polygon_offset_clamp GL_ARB_texture_filter_anisotropic GL_EXT_memory_object GL_EXT_memory_object_fd GL_EXT_semaphore GL_EXT_semaphore_fd GL_KHR_parallel_shader_compile GL_EXT_EGL_image_storage GL_EXT_shader_framebuffer_fetch_non_coherent GL_EXT_texture_shadow_lod GL_INTEL_blackhole_render GL_INTEL_shader_atomic_float_minmax GL_INTEL_shader_integer_functions2 GL_MESA_framebuffer_flip_y GL_NV_compute_shader_derivatives GL_EXT_EGL_sync GL_EXT_demote_to_helper_invocation
2022-10-07 09:12:58 [verbose]: Stored display index (-1) doesn't match current index (0)
2022-10-07 09:12:58 [verbose]: Detect: Searching for tablets...
2022-10-07 09:12:58 [verbose]: Updated display mode to desktop resolution: 1920x1080@144, SDL_PIXELFORMAT_RGB888
2022-10-07 09:12:58 [verbose]: Host execution state changed to Running
2022-10-07 09:12:58 [verbose]: Execution mode changed to MultiThreaded
2022-10-07 09:12:58 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:12:58 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:12:58 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:12:58 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:12:58 [verbose]: Detect: No tablets were detected.
2022-10-07 09:12:59 [verbose]: Beginning realm file store cleanup
2022-10-07 09:12:59 [verbose]: 🔈 BASS initialised
2022-10-07 09:12:59 [verbose]: BASS version:           2.4.16.7
2022-10-07 09:12:59 [verbose]: BASS FX version:        2.4.12.6
2022-10-07 09:12:59 [verbose]: BASS MIX version:       2.4.11.1
2022-10-07 09:12:59 [verbose]: Device:                 Default
2022-10-07 09:12:59 [verbose]: Driver:                 default
2022-10-07 09:12:59 [verbose]: Update period:          5 ms
2022-10-07 09:12:59 [verbose]: Device buffer length:   10 ms
2022-10-07 09:12:59 [verbose]: Playback buffer length: 100 ms
2022-10-07 09:12:59 [verbose]: Finished realm file store cleanup (0 of 2 deleted)
2022-10-07 09:12:59 [verbose]: Opened realm "/home/ivan/.var/app/sh.ppy.osu/data/osu/client.realm" at version 25
2022-10-07 09:12:59 [verbose]: Updated display mode to fullscreen resolution: 1920x1080@144, SDL_PIXELFORMAT_RGB888
2022-10-07 09:13:00 [verbose]: Loading FPSCounter...
2022-10-07 09:13:00 [verbose]: Loaded FPSCounter!
2022-10-07 09:13:00 [verbose]: Loading VersionManager...
2022-10-07 09:13:00 [verbose]: Loaded VersionManager!
2022-10-07 09:13:00 [verbose]: Loading OsuLogo...
2022-10-07 09:13:00 [verbose]: Loaded OsuLogo!
2022-10-07 09:13:00 [verbose]: Loading Toolbar...
2022-10-07 09:13:00 [verbose]: 📺 OsuScreenStack#389(depth:1) loading Loader#233
2022-10-07 09:13:00 [verbose]: 📺 OsuScreenStack#389(depth:1) entered Loader#233
2022-10-07 09:13:00 [verbose]: 📺 OsuScreenStack#389(depth:1) suspended Loader#233 (waiting on Disclaimer#543)
2022-10-07 09:13:00 [verbose]: 📺 OsuScreenStack#389(depth:2) entered Disclaimer#543
2022-10-07 09:13:00 [verbose]: Loaded Toolbar!
2022-10-07 09:13:00 [verbose]: Loading VolumeOverlay...
2022-10-07 09:13:00 [verbose]: Loaded VolumeOverlay!
2022-10-07 09:13:00 [verbose]: Loading OnScreenDisplay...
2022-10-07 09:13:00 [verbose]: Loaded OnScreenDisplay!
2022-10-07 09:13:00 [verbose]: Loading NotificationOverlay...
2022-10-07 09:13:00 [verbose]: Loaded NotificationOverlay!
2022-10-07 09:13:00 [verbose]: Loading LegacyImportManager...
2022-10-07 09:13:00 [verbose]: Loaded LegacyImportManager!
2022-10-07 09:13:00 [verbose]: Loading ScreenshotManager...
2022-10-07 09:13:00 [verbose]: Loaded ScreenshotManager!
2022-10-07 09:13:00 [verbose]: Loading NoActionUpdateManager...
2022-10-07 09:13:00 [verbose]: Loaded NoActionUpdateManager!
2022-10-07 09:13:00 [verbose]: Loading FirstRunSetupOverlay...
2022-10-07 09:13:00 [verbose]: Loaded FirstRunSetupOverlay!
2022-10-07 09:13:00 [verbose]: Loading ManageCollectionsDialog...
2022-10-07 09:13:01 [verbose]: Loaded ManageCollectionsDialog!
2022-10-07 09:13:01 [verbose]: Loading BeatmapListingOverlay...
2022-10-07 09:13:01 [verbose]: Loaded BeatmapListingOverlay!
2022-10-07 09:13:01 [verbose]: Loading DashboardOverlay...
2022-10-07 09:13:01 [verbose]: Loaded DashboardOverlay!
2022-10-07 09:13:01 [verbose]: Loading NewsOverlay...
2022-10-07 09:13:01 [verbose]: Loaded NewsOverlay!
2022-10-07 09:13:01 [verbose]: Loading RankingsOverlay...
2022-10-07 09:13:02 [verbose]: Loaded RankingsOverlay!
2022-10-07 09:13:02 [verbose]: Loading ChannelManager...
2022-10-07 09:13:02 [verbose]: Chat is now polling every 60000 ms
2022-10-07 09:13:02 [verbose]: Loaded ChannelManager!
2022-10-07 09:13:02 [verbose]: Loading ChatOverlay...
2022-10-07 09:13:02 [verbose]: Loaded ChatOverlay!
2022-10-07 09:13:02 [verbose]: Loading MessageNotifier...
2022-10-07 09:13:02 [verbose]: Loaded MessageNotifier!
2022-10-07 09:13:02 [verbose]: Loading SettingsOverlay...
2022-10-07 09:13:02 [verbose]: Loaded SettingsOverlay!
2022-10-07 09:13:02 [verbose]: Loading ChangelogOverlay...
2022-10-07 09:13:02 [verbose]: Loaded ChangelogOverlay!
2022-10-07 09:13:02 [verbose]: Loading UserProfileOverlay...
2022-10-07 09:13:02 [verbose]: Loaded UserProfileOverlay!
2022-10-07 09:13:02 [verbose]: Loading BeatmapSetOverlay...
2022-10-07 09:13:02 [verbose]: Loaded BeatmapSetOverlay!
2022-10-07 09:13:02 [verbose]: Loading WikiOverlay...
2022-10-07 09:13:02 [verbose]: Loaded WikiOverlay!
2022-10-07 09:13:02 [verbose]: Loading SkinEditorOverlay...
2022-10-07 09:13:02 [verbose]: Loaded SkinEditorOverlay!
2022-10-07 09:13:02 [verbose]: Loading LoginOverlay...
2022-10-07 09:13:02 [verbose]: Loaded LoginOverlay!
2022-10-07 09:13:02 [verbose]: Loading NowPlayingOverlay...
2022-10-07 09:13:02 [verbose]: Loaded NowPlayingOverlay!
2022-10-07 09:13:02 [verbose]: Loading AccountCreationOverlay...
2022-10-07 09:13:02 [verbose]: 📺 ScreenStack#412(depth:1) scheduling push ScreenWelcome#393
2022-10-07 09:13:02 [verbose]: Loaded AccountCreationOverlay!
2022-10-07 09:13:02 [verbose]: Loading DialogOverlay...
2022-10-07 09:13:02 [verbose]: Loaded DialogOverlay!
2022-10-07 09:13:02 [verbose]: Loading HighPerformanceSession...
2022-10-07 09:13:02 [verbose]: Loaded HighPerformanceSession!
2022-10-07 09:13:02 [verbose]: Loading BackgroundBeatmapProcessor...
2022-10-07 09:13:02 [verbose]: Beginning background beatmap processing..
2022-10-07 09:13:02 [verbose]: Querying for beatmap sets to reprocess...
2022-10-07 09:13:02 [verbose]: Loaded BackgroundBeatmapProcessor!
2022-10-07 09:13:02 [verbose]: Found 0 beatmap sets which require reprocessing.
2022-10-07 09:13:02 [verbose]: Querying for scores to reprocess...
2022-10-07 09:13:02 [verbose]: Found 0 scores which require reprocessing.
2022-10-07 09:13:02 [verbose]: Finished background beatmap processing!
2022-10-07 09:13:06 [verbose]: 📺 OsuScreenStack#389(depth:2) suspended Disclaimer#543 (waiting on IntroTriangles#617)
2022-10-07 09:13:06 [verbose]: 📺 OsuScreenStack#389(depth:3) entered IntroTriangles#617
2022-10-07 09:13:06 [verbose]: 📺 BackgroundScreenStack#532(depth:1) loading BackgroundScreenDefault#290
2022-10-07 09:13:06 [verbose]: Game-wide working beatmap updated to cYsmix - triangles (peppy) [peppy]
2022-10-07 09:13:06 [verbose]: 🌅 Background change queued
2022-10-07 09:13:06 [verbose]: 📺 BackgroundScreenStack#532(depth:1) entered BackgroundScreenDefault#290
2022-10-07 09:13:07 [verbose]: decoupled ruleset transferred ("" -> "osu!")
2022-10-07 09:13:10 [verbose]: 📺 OsuScreenStack#389(depth:3) suspended IntroTriangles#617 (waiting on MainMenu#205)
2022-10-07 09:13:10 [verbose]: 📺 OsuScreenStack#389(depth:4) entered MainMenu#205
2022-10-07 09:13:10 [verbose]: ⚠️ DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:13:10 [verbose]: ⚠️ DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:13:10 [verbose]: ⚠️ DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:13:10 [verbose]: ⚠️ Subsequent messages have been logged. Click to view log files.
2022-10-07 09:13:11 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2022-10-07 09:13:33 [verbose]: Chat is now polling every 600000 ms
2022-10-07 09:13:33 [verbose]: ButtonSystem's state changed from TopLevel to Initial
2022-10-07 09:13:36 [verbose]: Chat is now polling every 60000 ms
2022-10-07 09:13:37 [verbose]: ButtonSystem's state changed from Initial to TopLevel
2022-10-07 09:13:39 [verbose]: Focus contention triggered by NotificationOverlay.
2022-10-07 09:14:47 [verbose]: Tablet detected (vid1386), searching for usable configuration...
2022-10-07 09:14:47 [verbose]: Detect: Searching for tablets...
2022-10-07 09:14:47 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [error]: DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [verbose]: Detect: No tablets were detected.
2022-10-07 09:14:47 [verbose]: ⚠️ DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [verbose]: ⚠️ DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [verbose]: ⚠️ DeviceUnauthorizedAccessException: Not permitted to open HID class device at /dev/hidraw1.
2022-10-07 09:14:47 [verbose]: ⚠️ Subsequent messages have been logged. Click to view log files.
2022-10-07 09:14:57 [verbose]: Focus contention triggered by NotificationOverlay.
```

**Comment by jamesbt365** [comment]
https://opentabletdriver.net/Wiki/FAQ/Linux#fail-device-streams

**Comment by jamesbt365** [comment]
The message says you need dotnet 7.
